data/container-workload.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Container Workload
+After=opt-rootfs-resolv.service
+After=opt-rootfs-proc.mount
+
+[Service]
+RootDirectory=/opt/rootfs
+RootDirectoryStartOnly=yes
+Environment=PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+EnvironmentFile=-/opt/rootfs/.dockerenv
```

Comment:
Can you change this to `.containerenv` and update the `#define` for `CC_OCI_ENV_FILE` accordingly.